test/integration-test/commands/vc/verify-vc.test.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from '@oclif/test'
+import { vcPath } from '../../helpers/constants'
+
+describe('User verifies a VC', () => {
+  test
+    .stdout()
+    .command(['verify-vc', '-d', `${vcPath}`])
+    .it('verify  vc', (ctx) => {
+      expect(ctx.stdout).to.contains('isValid : true')
+      expect(ctx.stdout).to.contains('errors :')
```

Comment:
why is it supposed to contain an error ?
how would it be in case of a real error ?